test/csv/csv-filter/test_csv_filter.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: false
+
+require_relative '../helper'
+
+require 'csv'
+
+class TestFilter < Test::Unit::TestCase
+
+  # Some rows data (useful as default).
+  Rows = [
+    %w[aaa bbb ccc],
+    %w[ddd eee fff],
+  ]
+
+  # Return CSV string generated from rows array and options.
+  def make_csv_s(rows: Rows, **options)
+    CSV.generate(**options) do|csv|
+      rows.each do |row|
+        csv << row
+      end
+    end
+  end
```

Comment:
Do we need this?

How about keeping all CSV data as `String` not `[[...], [...], ...]` in this file?
I think that interface of `csv-filter` is CSV string not a CSV Ruby object.

Reply:
Done.